src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -25,33 +36,52 @@ export default {
     target_dataset_tooltip: T(''),
 
     recursive_placeholder: T('Recursive'),
-    recursive_tooltip: T(''),
+    recursive_tooltip: T('Replicate all child dataset snapshots.'),
 
     custom_snapshots_placeholder: T('Replicate Custom Snapshots'),
-    custom_snapshots_tooltip: T(''),
+    custom_snapshots_tooltip: T('Replicate snapshots that have not been \
+ created by an automated snapshot task. Requires defining the naming \
+ schema for the custom snapshots.'),
 
     naming_schema_placeholder: T('Naming Schema'),
-    naming_schema_tooltip: T(''),
+    naming_schema_tooltip: T('Custom snapshots to be replicated. Enter \
```

Comment:
It's not snapshots, but the way they are named.
```suggestion
    naming_schema_tooltip: T('Pattern of naming custom snapshots to be replicated. Enter \
```